container_images/test-manager/Dockerfile

```diff
@@ -0,0 +1,21 @@
+# Copyright 2020 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+FROM golang
+
+WORKDIR /go/src/github.com/GoogleCloudPlatform/guest-test-infra
+COPY . .
+RUN cd ./test_manager  && go get && CGO_ENABLED=0 GOOS=linux go build -v -o /test_manager
```

Comment:
i would usually run these as separate steps
this only compiles test_manager binary, we need to compile-in-place the test manager, the test wrapper, and every test package

Reply:
Split into multiple RUN command, add test wrapper and test